layers/generate_layers.py

```diff
@@ -71,6 +74,20 @@ def main():
         made_any = True
         makeCensusOutputAreas(args.census_output_areas)
 
+    if args.railway_stations:
+        made_any = True
+        generateLayerBasedOnTwoPartTag(
```

Comment:
The output files here retain all the OSM tags and substantially bloat file size. I think we just want the name for railway stations, so how about calling `remove_extra_properties` on the geojson?

Reply:
Also for railway stations, we only want nodes (aka points) where [railway=station](https://wiki.openstreetmap.org/wiki/Tag:railway%3Dstation). If any ways or relations happen to be tagged this way, it's a tagging error and we don't want to pick it up. So the "nwr" part of the osmium query just needs to be "n" for node. I think the helper function just hides a few of these problems; how about we have a normal method here that creates the temp dir, calls osmium, extracts to geojson, removes properties, and adds feature IDs?